README.md

```diff
@@ -17,7 +17,7 @@ up fast and efficiently at scale. Some of its features include:
   still preserving gRPC BadStatus codes
 * Server and client execution timings in responses
 
-gruf currently has active support for gRPC 1.10.x+. gruf is compatible and tested with Ruby 2.2-3.0.
+gruf currently has active support for gRPC 1.10.x+. gruf is compatible and tested with Ruby 2.2-3.1.
```

Comment:
Is `2.2` really accurate anymore if we're not actively testing it against that (or any other versions) previous to `2.6`?

Reply:
That's a good call, we should bump this to 2.6.